atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/TransactionManagerTest.java

```diff
@@ -137,9 +138,7 @@ public void shouldConflictIfImmutableTimestampLockExpiresEvenIfNoWritesOnThoroug
                 MultiTableSweepQueueWriter.NO_OP,
                 MoreExecutors.newDirectExecutorService());
 
-        when(timelock.getFreshTimestamp()).thenReturn(1L);
-        when(timelock.lockImmutableTimestamp(any())).thenReturn(
-                LockImmutableTimestampResponse.of(2L, LockToken.of(UUID.randomUUID())));
+        setupTimeLock(timelock);
```

Comment:
All of the setup above is also duplicated, so setup should take no arguments and return a `TransactionManager` instead.

Reply:
Well spotted! Changed.